### ISSUE TITLE
General questions about the "reallysimple" process...

### ISSUE THREAD

**Issue by scripting** [scripting/reallysimple#4]
I've been working for quite some time, a couple of years at least, on a set of building blocks for feed writing and reading, in time for the twenty-year anniversary of RSS 2.0 in September. Over the last couple of weeks, I've released code for the first lowest-level blocks. I imagine this is a bit hard to grasp at first. I'm here to explain, to answer questions.

**Comment by scripting** [comment]
One thing to look at is how the mailbox reader handles scripting news.

http://feeder.scripting.com/?template=mailbox&feedurl=http%3A%2F%2Fscripting.com%2Frss.xml

lots of decisions went into that. works the same for all drummer blogs, so anyone who's using drummer to write their blog can have a look might be interesting.

**Comment by scripting** [comment]
Something I've been meaning to do -- seeing how a micro.blog feed looks in the mailbox template.

http://feeder.scripting.com/?template=mailbox&feedurl=http://dave.micro.blog/feed.xml

**Comment by rhempel** [comment]
I've written the guts of reallysimple in Python - currently it takes the NY Times World Headline URL in your example and pushes out a JSONified string of just the items in the feed - with the names adjusted to match the ones in https://github.com/scripting/reallysimple/blob/main/example/test.json

I can build the header for the feed tomorrow and make the repo available publicly - it uses the feedparser package available on PyPi to parse the actual feed into a Python struct - the resolved dictinary names are slightly different than your allowed names, so I made a simple Python dict to map the feedparser names to yours.

Long story short, the port is pretty easy - I guess the use case isn't quite clear to me - unless it's just breaking a feed down into a format that other apps that use the reallysimple format can use.
